docs/usage/index.md

```diff
@@ -7,7 +7,7 @@ sidebar_position: 3
 ## Debugging deployments using BotKube
 
 @BotKube allows you to execute kubectl commands on your Kubernetes cluster.
-Run **/botkubehelp** to find more information about the supported commands.
```

Comment:
See also the `Managing notifications` section and fix it pls as well - there is also the `/botkubehelp` command mentioned.